test/parse_tests.c

```diff
@@ -65,6 +65,8 @@ const char* tests[] = {
   "{\"version\":150001,\"stmts\":[{\"stmt\":{\"AlterTableMoveAllStmt\":{\"orig_tablespacename\":\"foo\",\"objtype\":\"OBJECT_TABLE\",\"roles\":[{\"RoleSpec\":{\"roletype\":\"ROLESPEC_CSTRING\",\"rolename\":\"bar\",\"location\":43}},{\"RoleSpec\":{\"roletype\":\"ROLESPEC_CSTRING\",\"rolename\":\"quux\",\"location\":48}}],\"new_tablespacename\":\"fred\",\"nowait\":true}}}]}",
   "CREATE PUBLICATION foo FOR TABLES IN SCHEMA bar",
   "{\"version\":150001,\"stmts\":[{\"stmt\":{\"CreatePublicationStmt\":{\"pubname\":\"foo\",\"pubobjects\":[{\"PublicationObjSpec\":{\"pubobjtype\":\"PUBLICATIONOBJ_TABLES_IN_SCHEMA\",\"name\":\"bar\",\"location\":44}}]}}}]}",
+  "SELECT 123abc",
```

Comment:
+1 to adding a test here, I wonder if we should use an example that includes param refs as well? (or just have that)

Anticipating that in Postgres 16 we'd not support this syntax for regular integers, but would keep supporting the syntax for param refs.

Reply:
Have added a test for `SELECT 123 AS abc where 456=$1OR 10=11` 👍